<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Terminal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionOpenProg"/>
    <addaction name="actionSaveProg"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionSystem"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenProg"/>
   <addaction name="actionSaveProg"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionSystem"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="tuner.qrc">
     <normaloff>:/images/help_48dp.png</normaloff>:/images/help_48dp.png</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="toolTip">
    <string>帮助</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="tuner.qrc">
     <normaloff>:/images/link.png</normaloff>:/images/link.png</iconset>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
   <property name="toolTip">
    <string>连接</string>
   </property>
   <property name="statusTip">
    <string>开启串口通信</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="tuner.qrc">
     <normaloff>:/images/link_off.png</normaloff>:/images/link_off.png</iconset>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
   <property name="toolTip">
    <string>断开</string>
   </property>
   <property name="statusTip">
    <string>断开串口通信</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="tuner.qrc">
     <normaloff>:/images/serial.png</normaloff>:/images/serial.png</iconset>
   </property>
   <property name="text">
    <string>串口配置</string>
   </property>
   <property name="toolTip">
    <string>串口配置</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="tuner.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionSystem">
   <property name="icon">
    <iconset resource="tuner.qrc">
     <normaloff>:/images/settings_new.png</normaloff>:/images/settings_new.png</iconset>
   </property>
   <property name="text">
    <string>系统配置</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionOpenProg">
   <property name="icon">
    <iconset resource="tuner.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开程序</string>
   </property>
  </action>
  <action name="actionSaveProg">
   <property name="icon">
    <iconset resource="tuner.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存程序</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tuner.qrc"/>
 </resources>
 <connections/>
</ui>
